enry.go

```diff
@@ -1,3 +1,17 @@
 package enry // import "gopkg.in/src-d/enry.v1"
 
 //go:generate make code-generate
+
+/*
```

Comment:
In order for this to count as the package comment, it must immediately precede the `package` clause.